exports.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
         'acmValidation'                 : require(__dirname + '/plugins/aws/acm/acmValidation.js'),
         'acmCertificateExpiry'          : require(__dirname + '/plugins/aws/acm/acmCertificateExpiry.js'),
         'asgMultiAz'                    : require(__dirname + '/plugins/aws/autoscaling/asgMultiAz.js'),
+        'emptyAsg'                      : require(__dirname + '/plugins/aws/autoscaling/emptyAsg.js'),
```

Comment:
you named the file: `emptyASG.js` and here you put `emptyAsg.js`. This is why this PR is failing the checks